README.md

```diff
@@ -49,6 +49,8 @@ You can also use dynamic values from the datastore. See the
 
 * ``create_resource`` - Create a generic ARM resource
 * ``create_linked_resource_url`` - Create a linked (template and parameter) resource from a URI
+* ``list_resource_groups`` - List the names of the resource groups
+* ``default_resource_group`` - Default resource group
```

Comment:
That section is a list of actions - I'm not sure that `default_resource_group` belongs there?